scripts/js_fixes.sh

```diff
@@ -1,3 +1,3 @@
 export OUT_DIR=gen/js
 
-protoc -I./deps/googleapis deps/googleapis/google/type/* deps/googleapis/google/api/* -I./deps/protoc-gen-openapiv2 deps/protoc-gen-openapiv2/options/* --js_out=import_style=commonjs:$OUT_DIR
+protoc -I./deps/googleapis deps/googleapis/google/type/* deps/googleapis/google/api/* -I./deps/protoc-gen-openapiv2 deps/protoc-gen-openapiv2/options/* -I./deps/validation deps/validation/validate/* --js_out=import_style=commonjs:$OUT_DIR
```

Comment:
I'm still kinda puzzled why is this needed. I get that imports are not working without all of its dependencies but I still think server-side validation shouldn't require to have validation import for client-side code

Reply:
For future reference - this seems to be a general problem with the validation library. For now, one can only decide to either include the validation library in the javascript generated files or do some text replacing on the generated files afterwards. See [this stackoverflow post](https://stackoverflow.com/questions/62983718/how-to-exclude-an-import-on-client-side) for example.